### DISCUSSION TITLE
Value is empty on initial render and sets after few milliseconds

### DISCUSSION THREAD

**Discussion by mahavirn** [pmndrs/zustand#2408] [Bug report]
## Summary
Value is empty on initial render and sets after few milliseconds

const user = globalStore((state) => state.user);

if i console.log(user) its {} initially and after 100-200 ms the value is resolved to actual value.

**Comment by dai-shi** [comment]
can you create a small reproduction with https://livecodes.io/ or https://stackblitz.com/ ?
